charts/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: grafana
-version: 7.0.1
+version: 7.0.2
```

Comment:
I consider this a bug fix, so I bumped the patch version.

Reply:
I'm sorry but this would be classified as a breaking change and as such a patch bump is not sufficient.